homeassistant/components/lcn/const.py

```diff
@@ -16,11 +16,47 @@
 CONF_DIMMABLE = 'dimmable'
 CONF_TRANSITION = 'transition'
 CONF_MOTOR = 'motor'
+CONF_SOURCE = 'source'
 
 DIM_MODES = ['STEPS50', 'STEPS200']
+
 OUTPUT_PORTS = ['OUTPUT1', 'OUTPUT2', 'OUTPUT3', 'OUTPUT4']
+
 RELAY_PORTS = ['RELAY1', 'RELAY2', 'RELAY3', 'RELAY4',
                'RELAY5', 'RELAY6', 'RELAY7', 'RELAY8',
                'MOTORONOFF1', 'MOTORUPDOWN1', 'MOTORONOFF2', 'MOTORUPDOWN2',
                'MOTORONOFF3', 'MOTORUPDOWN3', 'MOTORONOFF4', 'MOTORUPDOWN4']
+
 MOTOR_PORTS = ['MOTOR1', 'MOTOR2', 'MOTOR3', 'MOTOR4']
+
+LED_PORTS = ['LED1', 'LED2', 'LED3', 'LED4', 'LED5', 'LED6',
+             'LED7', 'LED8', 'LED9', 'LED10', 'LED11', 'LED12']
+
+LOGICOP_PORTS = ['LOGICOP1', 'LOGICOP2', 'LOGICOP3', 'LOGICOP4']
+
+VARIABLES = ['VAR1ORTVAR', 'VAR2ORR1VAR', 'VAR3ORR2VAR',
+             'TVAR', 'R1VAR', 'R2VAR',
+             'VAR1', 'VAR2', 'VAR3', 'VAR4', 'VAR5', 'VAR6',
+             'VAR7', 'VAR8', 'VAR9', 'VAR10', 'VAR11', 'VAR12']
+
+SETPOINTS = ['R1VARSETPOINT', 'R2VARSETPOINT']
+
+THRESHOLDS = ['THRS1', 'THRS2', 'THRS3', 'THRS4', 'THRS5',
+              'THRS2_1', 'THRS2_2', 'THRS2_3', 'THRS2_4',
+              'THRS3_1', 'THRS3_2', 'THRS3_3', 'THRS3_4',
+              'THRS4_1', 'THRS4_2', 'THRS4_3', 'THRS4_4']
+
+S0_INPUTS = ['S0INPUT1', 'S0INPUT2', 'S0INPUT3', 'S0INPUT4']
+
+VAR_UNITS = ['', 'LCN', 'NATIVE',
+             'CELSIUS', '\u00b0CELSIUS', '\u00b0C',
```

Comment:
We standardize some units like eg temperature units (`TEMP_CELSIUS`, `TEMP_FAHRENHEIT`). See const.py.

Reply:
Thanks for the review!
I changed the temperature units to the standards. See the PR here: #22108
Once there are more stardadized units in Home Assistant, the code might need some rework to also support them. I will have a look after that.
I also updated the documentation accordingly: #8734